copyright

```diff
@@ -545,6 +632,20 @@ Copyright: Becca Tommaso (tommasobecca03@gmail.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license).
 
+Files:
+ images/outfit/*battery?hai*
+ images/outfit/anti-missile?hai*
+ images/outfit/cooling?ducts?hai*
+ images/outfit/dwarf?core?hai*
+ images/outfit/fission?hai*
+ images/outfit/fusion?hai*
+ images/outfit/heavy?anti-missile?hai*
+ images/ship/mfury*
+ images/ship/mquicksilver*
```

Comment:
Should be in a different place:

https://github.com/endless-sky/endless-sky/blob/941639dc6f7a2abdb945a7b45740e77f9dbc9c32/copyright#L1032-L1041

The corresponding high-DPI block is:

https://github.com/endless-sky/endless-sky-high-dpi/blob/bc3f6f51ae4f64cfcbcc65ab0a51ee3464cafd0e/copyright#L1068-L1075